src/utils.js

```diff
@@ -198,21 +199,14 @@ export function isObject(thing) {
   return typeof thing === 'object' && thing !== null && !Array.isArray(thing);
 }
 
+function mergeArrays(objValue, srcValue) {
```

Comment:
Should this function only return something if an array exists?

Reply:
If `objValue` is an array the function will concat `srcValue` to `objValue`. If `objValue` is not an array `srcValue` will override `objValue`.
I'll make a fix to perform the concat if and only if both values are arrays.

Reply:
We're discussing a offline about this and trying a different approach.

Reply:
We decided to add a simple check at the beginning of the function to check the value of `obj1`